cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -186,13 +214,16 @@ public String getId() {
     }
 
     public CgmesExportContext() {
+        // FIXME(Luma) review how this is used, no default constructor should be allowed
+        this.network = null;
```

Comment:
This is used to export merging view: we don't want the export to be built from the merging view but from the extensions of the different networks. In this case you can't set `network` to `null`. I actually don't think network should be in the context... Why this design? We should pass the network to retrieve information/data when necessary, but not store it  in my opinion.